velox/functions/prestosql/types/BigintEnumType.h

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <boost/algorithm/string.hpp>
+
+#include <map>
+#include "folly/CPortability.h"
+#include "velox/type/Type.h"
+
+namespace facebook::velox {
+
+class BigintEnumType : public BigintType {
```

Comment:
Please, review coding style and update accordintly.

All APIs in header files must be documented.

Implementation of methods to into .cpp file.